### PR TITLE
Move OpenAPI validation to a GitHub action.

### PR DESCRIPTION
**Pull request by paolino**
- [x] move the openapi validity check to a single github action

### CODE REVIEW

File: .github/workflows/openapi.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: API Quality
```
Comment (Anviking) [inline .github/workflows/openapi.yml]: Perhaps just `OpenAPI spec validation` or something similar would be clearer?

**Review by Anviking — APPROVED** [review]

File: .github/workflows/openapi.yml (hunk 1 above)
Comment (paolino) [reply .github/workflows/openapi.yml]: I was inspired by "Coding Standards" of the other actions :-) . But in reality we have only one check here .. changing it

**Review by jonathanknowles — APPROVED** [review]
Nice work @paolino!

Screenshot 2022-12-08 at 19-41-33 remove openapi check from buildkite · input-output-hk_cardano-wallet@16de33d

**Review by paweljakubas — APPROVED** [review]
lgtm!

**Comment by paolino** [comment]
we stick to buildkite

**Pull request closed without merging**